types/stylelint/index.d.ts

```diff
@@ -375,7 +375,23 @@ declare namespace stylelint {
 		cwd: string;
 		results: LintResult[];
 		errored: boolean;
+		/**
+		 * @deprecated Use `output` for the formatted problems, or use `code`
+		 *   for the autofixed code instead. This will be removed in the next major version.
+		 */
```

Comment:
```suggestion
		/**
		 * @deprecated Use `report` for the formatted problems, or use `code`
		 *   for the autofixed code instead. This will be removed in the next major version.
		 */
```